samples/python/test_samples.py

```
@@ -0,0 +1,127 @@
+import pytest
+
+SAMPLES_PYTHON_DIR = Path(__file__).parent
+EXCLUDE_DIRS = {"Utilities", "__pycache__"}
+EXIT_WAIVED = 2
+
+
+def _cuda_compute_importable() -> bool:
+    """True when the `cuda.compute` module is available in this env."""
+    try:
+        import cuda.compute  # noqa: F401
+    except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)